tests_e2e/tests/lib/ssh_client.py

```diff
@@ -52,10 +57,19 @@ def generate_ssh_key(private_key_file: Path):
     def get_architecture(self):
         return self.run_command("uname -m").rstrip()
 
-    def copy(self, source: Path, target: Path, remote_source: bool = False, remote_target: bool = False, recursive: bool = False):
```

Comment:
This interface was a little hard to use, so I split this method into copy_to_node and copy_from_node